tests/foreman/api/test_hostcollection.py

```diff
@@ -392,17 +391,22 @@ def test_negative_create_with_invalid_name(self):
                     entities.HostCollection(name=name, organization=self.org).create()
 
     @tier1
-    def test_positive_add_subscription(self):
-        """Try to bulk add a subscription to members of a host collection.
+    def test_positive_add_remove_subscription(self):
+        """Try to bulk add and remove a subscription to members of a host collection.
 
         :id: c4ec5727-eb25-452e-a91f-87cafb16666b
 
         :steps:
 
-            1. Create a new or use an existing subscription
-            2. Add the subscription to the members of the host collection
+            1. Create AK and HC, add AK to HC
+            2. Create product so we can use it's subscription
+            3. Create some VMs and register them with AK so they are in HC
+            2. Add the subscription to the members of the Host Collection
+            3. Assert subscription is added
```

Comment:
the numbering here probably makes make_docstrings fail in travis

Reply:
oh dear! fix made, thank you for spotting that @pondrejk